src/components/TopBarMenu/User/User.jsx

```diff
@@ -0,0 +1,7 @@
+import "./User.scss";
+
+export default function User({ userSource }) {
+  const IMAGE_PATH = "/images/";
+
+  return <img className="user-icon" src={IMAGE_PATH + userSource} alt="" />;
+}

```

Comment:
```suggestion
export default function User({ userSource, alt = "" }) {
  const IMAGE_PATH = "/images/";

  return <img className="user-icon" src={IMAGE_PATH + userSource} alt={alt} />;
}
```

it's a best practice to pass an alt for all images:

> The ALT text adds a text description to an image on a Web page, and should be used for all images, graphical bullets, and graphical horizontal rules. ALT text is accessed by screen reader users to provide them with a text equivalent of images.

https://accessibility.psu.edu/images/imageshtml/#:~:text=The%20ALT%20text%20adds%20a,a%20text%20equivalent%20of%20images.